pkg/users/authorized-keys-panel.js

```diff
@@ -91,73 +96,54 @@ export function AuthorizedKeys({ name, home, allow_mods }) {
-                        {_("There are no authorized public keys for this account.")}
-                    </div>
-                </li>
-            ];
-        } else {
-            key_items = keys.map(k =>
-                <li key={k.raw} className="pf-c-data-list__item">
-                    <div className="pf-c-data-list__item-row">
-                        <div className="pf-c-data-list__item-content">
-                            <div className="pf-c-data-list__cell comment">
-                                { k.comment || <em>{_("Unnamed")}</em> }
-                            </div>
-                            <div className="pf-c-data-list__cell fingerprint">
-                                { k.fp || <span>{_("Invalid key")}</span> }
-                            </div>
-                        </div>
-                        { allow_mods &&
-                        <div className="pf-c-data-list__item-action">
-                            <Button variant="secondary" onClick={() => remove_key(k.raw)}
-                                    className="account-remove-key">
-                                {_("Remove")}
-                            </Button>
-                        </div> }
-                    </div>
-                </li>);
-        }
-    } else
+    let error = "";
+
+    if (state == "access-denied")
+        error = _("You do not have permission to view the authorized public keys for this account.");
+    else if (state == "failed")
+        error = _("Failed to load authorized keys.");
+    else if (state == "ready")
+        error = _("There are no authorized public keys for this account.");
+    else
         return null;
 
     return (
-        <div className="pf-c-card" id="account-authorized-keys">
-            <div className="pf-c-card__header">
-                <div className="pf-c-card__title"><h2>{_("Authorized public SSH keys")}</h2></div>
+        <Card id="account-authorized-keys">
+            <CardHeader>
+                <CardTitle><Text component={TextVariants.h2}>{_("Authorized public SSH keys")}</Text></CardTitle>
                 { allow_mods &&
-                <Button onClick={() => add_authorized_key_dialog(manager)}
-                        id="authorized-key-add">
-                    {_("Add key")}
-                </Button>}
-            </div>
-            <div className="pf-c-card__body contains-list">
-                <ul className="pf-c-data-list pf-m-compact" id="account-authorized-keys-list">
-                    { key_items }
-                </ul>
-            </div>
-        </div>
+                <CardActions>
+                    <Button variant="secondary" id="authorized-key-add" onClick={() => add_authorized_key_dialog(manager)}>
+                        {_("Add key")}
+                    </Button>
+                </CardActions>
+                }
+            </CardHeader>
+            <ListingTable
```

Comment:
This breaks mobile. I can help you if you want to make it work with this code.

Reply:
Do we need some breakpoints there?

Reply:
ah I see, I am taking a look

Reply:
I found two similar PF examples.  Basically they convert the button to kebab menu on mobile view. Maybe you could try it

previews:
https://www.patternfly.org/v4/components/table/react-composable/composable-actions-overflow/
https://www.patternfly.org/v4/components/table/react-composable/composable-actions/

code here: https://www.patternfly.org/v4/components/table/#composable-actions-overflow

Reply:
What @tomasmatus suggests or use DataList instead of ListingTable.